bedrock/mozorg/templates/mozorg/mission_ftl.html

```diff
@@ -0,0 +1,77 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/about-base.html" %}
+
+{% block page_title %}{{ ftl('mission_were_building_a_better_internet') }}{% endblock %}
+{% set body_id = "about-mission" %}
+
+{% block page_css %}
+  {{ css_bundle('mission') }}
+{% endblock %}
+
+{% block article %}
+<article>
+  <h1 class="mzp-c-article-title">{{ ftl('mission_were_building_a_better_internet') }}</h1>
+  {% if l10n_has_tag('bug_1251061_copy_update') %}
+    <p class="lede">{{ ftl('mission_our_mission_is_to_ensure_the_internet') }}</p>
```

Comment:
@alexgibson this template was auto-converted by my new management commands. I'd love to hear your thoughts.

Reply:
I'll take this for a spin and report back 👍